mlflow/sklearn/__init__.py

```diff
@@ -481,7 +481,7 @@ def __init__(self, sk_model, original_exception):
 
 def _dump_model(pickle_lib, sk_model, out):
     try:
-        pickle_lib.dump(sk_model, out)
+        pickle_lib.dump(sk_model, out, protocol=pickle.DEFAULT_PROTOCOL)
```

Comment:
Can we add a comment explaining why we set `protocol`?

Reply:
Done.